koku/api/resource_types/test/aws_category/test_views.py

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright 2021 Red Hat Inc.
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Test the Resource Types views."""
+import logging
+
+from django.urls import reverse
+from rest_framework import status
+from tenant_schemas.utils import schema_context
+
+from api.iam.test.iam_test_case import RbacPermissions
+from api.utils import DateHelper
+from masu.test import MasuTestCase
+from reporting.provider.aws.models import AWSEnabledCategoryKeys
+
+LOG = logging.getLogger(__name__)
+
+
+class ResourceTypesViewTestAWSCategory(MasuTestCase):
+    """Tests the resource types views."""
+
+    @classmethod
+    def setUpClass(cls):
+        """Set up the customer view tests."""
+        super().setUpClass()
+
+    def setUp(self):
+        """Set up a test with database objects."""
+        super().setUp()
+        self.dh = DateHelper()
+        self.start_date = self.dh.this_month_start
+        self.end_date = self.dh.this_month_end
+
+    @RbacPermissions({"aws.account": {"read": ["*"]}})
+    def test_aws_categories_view_with_wildcard(self):
+        """Test aws categories return."""
+        with schema_context(self.schema):
+            expected = AWSEnabledCategoryKeys.objects.filter(enabled=True).values_list("key", flat=True)
+            url = reverse("aws-categories")
+            response = self.client.get(url, **self.headers)
+            self.assertEqual(response.status_code, status.HTTP_200_OK)
+            json_result = response.json()
+            LOG.info(json_result)
```

Comment:
We don't need to log anything in unit tests, so let's remove these and the logger.

Reply:
Same as above, your comment lead me down an audit and I removed the logger from the unittest files.